libs/styles/src/lib/scss/components/_steps.scss

```diff
@@ -99,6 +103,11 @@
         
       }
       @include cvi-breakpoint-up(sm) {
+        #{$base}.is-scrollable & {
+          min-width: 247px;
+          height: 72px;
```

Comment:
Height should not be hardcoded (there should be no limit on the maximum height).

Reply:
Yes, I could fix it. Explanation in [this comment](https://github.com/e-gov/cvi/pull/227#discussion_r1590772137)